src/twisted/application/internet.py

```diff
@@ -831,8 +841,8 @@ def _unawait(self, value):
     _init.upon(start, enter=_connecting,
                outputs=[_connect])
     _init.upon(stop, enter=_stopped,
-               outputs=[],
-               collector=lambda _: succeed(None))
+               outputs=[_deferredSucceededWithNone],
```

Comment:
This change doesn't seem to add any clarity to what the code is doing, and add more code that needs to be reasoned about.